install.sh

```diff
@@ -169,6 +169,9 @@ if [ "$(uname)" = "Linux" ]; then
   fi
 elif [ "$(uname)" = "Darwin" ] && ! type brew >/dev/null 2>&1; then
   echo "Installation currently requires brew on MacOS - https://brew.sh/"
+  exit 1
+elif [ "$(uname)" = "Darwin" ]; then
+  echo "Installing OpenSSL"
   brew install openssl
```

Comment:
Add the 'installing on' notification regardless, and group the macOS code all together in one `if`.  I don't know that I have this exactly right, coded off the cuff and untested.

```suggestion
elif [ "$(uname)" = "Darwin" ]; then
  echo "Installing on macOS."
  if ! type brew >/dev/null 2>&1; then
    echo "Installation currently requires brew on macOS - https://brew.sh/"
    exit 1
  fi
  echo "Installing OpenSSL"
  brew install openssl
```